pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
> Yeah I think the wrapper that makes sense and models what we'd do for end
> users.  Ref to image, although we may want to make the signature name be
> the unique verification (I don't know that I want a special name for a
> signature).

That makes sense.  Does the `RESTHandler` include the requested name part of the URL anywhere in the context to avoid having to repeat myself in the API?